src/Cubes/Cubes.jl

```diff
@@ -81,8 +81,9 @@ struct YAXArray{T,N,A<:AbstractArray{T,N},AT}
     axes::AT
     data::A
     properties::Dict{String}
+    chunks::GridChunks{N}
     cleaner::Vector{CleanMe}
-    function YAXArray(axes, data, properties, cleaner)
+    function YAXArray(axes, data, properties, chunks, cleaner)
```

Comment:
Isn't this change breaking? Because I can't do `YAXArray(axes, data, properties ,cleaner)` anymore.